### PR TITLE
Add Harvest, Issuance modules to swagger

### PR DESCRIPTION
**Pull request by denalimarsh**
The previous swagger update just updated the CDP/Auction modules - this PR adds Harvest and Issuance posts and queries to the swagger documentation.

We still need to update the swagger docs for Cosmos SDK v0.39. Is updating the account number/sequence data types sufficient? @karzak you mentioned that the JSON structure had also changed. Will this require additional updates?

### CODE REVIEW

File: swagger-ui/swagger.yaml [hunk 1]
```diff
@@ -1696,6 +1700,436 @@
             description: Invalid query parameters
           500:
             description: Internal Server Error
+    /harvest/deposit:
+      post:
+        summary: Deposit funds to harvest liquidity pool
+        tags:
+          - harvest
```
Comment (karzak) [inline swagger-ui/swagger.yaml:1707]:
```suggestion
          - Harvest
```

File: swagger-ui/swagger.yaml [hunk 2]
```diff
@@ -1696,6 +1700,436 @@
...
+              $ref: '#/definitions/StdTx'
+          400:
+            description: Invalid request
+          500:
+            description: Internal server error
+    /harvest/withdraw:
+      post:
+        summary: Withdraw funds from harvest liquidity pool
+        tags:
+          - harvest
```
Comment (karzak) [inline swagger-ui/swagger.yaml:1739]: ditto

File: swagger-ui/swagger.yaml [hunk 3]
```diff
@@ -1696,6 +1700,436 @@
...
+              $ref: '#/definitions/StdTx'
+          400:
+            description: Invalid request
+          500:
+            description: Internal server error
+    /harvest/claim:
+      post:
+        summary: Claim rewards from harvest liquidity pool
+        tags:
+          - harvest
```
Comment (karzak) [inline swagger-ui/swagger.yaml:1771]: ditto

**Review by karzak — APPROVED** [review]
Fix tags so the harvest commands all group together. Otherwise LGTM

**Comment by karzak** [comment]
> We still need to update the swagger docs for Cosmos SDK v0.39. Is updating the account number/sequence data types sufficient? @karzak you mentioned that the JSON structure had also changed. Will this require additional updates?

The serialization is the only thing that changed. Accounts number and sequence are now serialized to strings - they examples already have them that way so I think we're fine.

File: swagger-ui/swagger.yaml (hunk 3 above)
Comment (denalimarsh) [reply swagger-ui/swagger.yaml]: https://github.com/Kava-Labs/kava/pull/670/commits/9dec028708bff2c2a439b41f591a736b518d3f06

File: swagger-ui/swagger.yaml (hunk 2 above)
Comment (denalimarsh) [reply swagger-ui/swagger.yaml]: https://github.com/Kava-Labs/kava/pull/670/commits/9dec028708bff2c2a439b41f591a736b518d3f06

File: swagger-ui/swagger.yaml (hunk 1 above)
Comment (denalimarsh) [reply swagger-ui/swagger.yaml]: Updated in https://github.com/Kava-Labs/kava/pull/670/commits/9dec028708bff2c2a439b41f591a736b518d3f06

**Pull request merged**